astropy/nddata/tests/test_utils.py

```diff
@@ -316,3 +320,87 @@ def test_block_size_len(self):
         data = np.arange(5)
         with pytest.raises(ValueError):
             block_replicate(data, (2, 2))
+
+
+class TestCutout(object):
+    data = np.arange(20.).reshape(5, 4)
```

Comment:
I think I've never seen this way to set up test data ... is it OK? Is there a possibility that one test modifies this (explicitly or by accident) and then the tests are connected?

I think the recommended practice is fixtures or `setup_class` classmethods?
http://pytest.org/latest/xunit_setup.html#class-level-setup-teardown
https://github.com/astropy/astropy/blob/5d419bbc438701e3bc99819ae5a36d9de95f5d41/astropy/vo/validator/tests/test_inpect.py#L21